doc/extensions/acrbuildext/azext_acrbuildext/build.py

```diff
@@ -244,8 +244,7 @@ def acr_queue(cmd,
 
     # hard-code platform to linux and cpu to 1
```

Comment:
Platform may also be removed. You can leave it for this PR. BTW: It's good we did the test and verify the code path exposing the service-side issue =)

Reply:
I'll make a separate issue on this GitHub so we can track it. Merging for now.